### DISCUSSION TITLE
Is there an estimated timeline for SIP12?

### DISCUSSION THREAD

**Discussion by ben-sid** [MetaMask/snaps#1872] [Q&A] [answered]
I believe this snaps improvements proposals re opening up Domain Resolution will be quite a meaningful improvement. Is there an estimated timeline for when will SIP12 be final/live? 

https://metamask.github.io/SIPs/SIPS/sip-12

Many thanks.
Ben

**Comment by Montoya** [comment]
Hi! We are planning to ship the first iteration of this feature in Flask 11.4, which will be available by November. I think the SIP will still be in draft form while we are testing the API with developers.

**Reply by ben-sid** [reply]
Got it. Thanks!

Look forward to it!

**Reply by ben-sid** [reply]
Hi Montoya, is there somewhere in Flask 11.4 that we can play around the Domain Resolution feature?

**Reply by Montoya** [reply, marked as answer]
Not yet. It's still pending merge: https://github.com/MetaMask/metamask-extension/pull/19131

You can use the Flask build from that thread to play around with the feature.

**Reply by ben-sid** [reply]
Kk, thx!

**Reply by ben-sid** [reply]
Hi Montoya,

We have tried out this domain resolution Snap on Flask - Great experience!

Wondering what's the next step? When will this be live on MetaMask for us to submit the Snap?
